index.html

```diff
@@ -76,7 +76,7 @@ <h2 hidden>Game over</h2>
     )
   )
   const gameOver = document.querySelector('h2')
-  const HUMAN = '🧑‍💻'
```

Comment:
Hi, a guess!! May be cleaner if the computer icon is deleted if it's not needed.Cheers.